module/batch-action/src/Infrastructure/Persistence/Query/DbalBatchActionQuery.php

```diff
@@ -73,11 +73,12 @@ public function getProfileInfo(): array
     {
         $qb = $this->connection->createQueryBuilder();
 
-        return $qb->select('id')
+        return  $qb->select('id')
             ->addSelect('(select (case
-                                            when (select bool_and(success)
+                                            when (select count(*)
                                                   from batch_action_entry
-                                                  where batch_action_id = ba.id) then \'ENDED\'
+                                                  where batch_action_id = ba.id 
+                                                    and success is not null ) = 0 then \'ENDED\'
                                             else \'PRECESSED\' end) as status)')
             ->addSelect('created_at as started_at')
             ->addSelect('(select MAX(processed_at)
```

Comment:
Are we delimiting the results by the fact of having any entries? I guess we should if we do not - since we can have action which do not have entries created just yet.